src/components/markdown/references/references.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { useContext } from 'react'
+import PropTypes, { array } from 'prop-types'
+
+import { BibReferencesContext } from '../../page-template'
+
+const References = ({ children, citationStyle }) => {
+  const { references, usedReferences } = useContext(
+    BibReferencesContext
+  )
+
+
+  // looping array
+  const referenceOptions = Array.from(usedReferences).map((referenceId, i) => {
+
+    const fields = references[referenceId].fields     // fields handles all the data you need, e.g. author, title, year, etc
+    const types = references[referenceId].type        // types handles the type of bibtex reference entry
+
+    const authorName = fields.author.split(' ');    // splitting author name for future use
+
+    // APA STYLING
+    if(citationStyle == "APA")
```

Comment:
We can probably do with just one citation style: IEEE seems like a good option. [This article](https://guides.lib.monash.edu/citing-referencing/ieee) shows how to do it. [And this](https://vimeo.com/220916942) is also a nice video.

So we can just have the one style and remove the others. Keep the separate format for each type of reference, like you've done below with APA, it's nice 👍